book/text.md

```diff
@@ -217,8 +213,6 @@ of course, since different letters have different widths:[^widths]
 5
 >>> bi_times.measure("!")
 7
->>> 13 + 5 + 7
-25
```

Comment:
Arguable, but we only refer to this sum in a footnote, so perhaps it doesn't need to be in the main text?